packages/react-geo/src/styles.css

```diff
@@ -1 +1,4 @@
 @import '../../ui/dist/styles.css';
+@import '@maplibre/maplibre-gl-geocoder/dist/maplibre-gl-geocoder.css';
+@import 'maplibre-gl-js-amplify/dist/public/amplify-geocoder.css';
+@import 'maplibre-gl/dist/maplibre-gl.css';
```

Comment:
Can we order the imports so that it's:

1. 3p deps (`maplibre-gl`, `@maplibre`)
2. Internal dependencies (`maplibre-gl-js-amplify`)
3. Local dependencies (`../../ui/dist/...`)